web/html/src/.storybook/preview-head.html

```diff
@@ -0,0 +1,25 @@
+<!-- import default fonts/icons styles -->
+<link rel="stylesheet" href="font-awesome.css?cb=${cb_version}" />
+<!-- import custom fonts/icons styles -->
+<link rel="stylesheet" href="https://head-server.tf.local/fonts/font-spacewalk/css/spacewalk-font.css?cb=${cb_version}" />
+
+<!-- import plugins styles -->
+<link rel="stylesheet" href="https://head-server.tf.local/css/jquery.timepicker.css?cb=${cb_version}" />
+<link rel="stylesheet" href="https://head-server.tf.local/css/bootstrap-datepicker.css?cb=${cb_version}" />
+<link rel="stylesheet" href="https://head-server.tf.local/javascript/select2/select2.css?cb=${cb_version}" />
+<link rel="stylesheet" href="https://head-server.tf.local/javascript/select2/select2-bootstrap.css?cb=${cb_version}" />
+<link rel="stylesheet" href="https://head-server.tf.local/css/spacewalk.css?cb=${cb_version}" />
+
+<script src="https://head-server.tf.local/javascript/loggerhead.js?cb=${cb_version}"></script>
+<script src="https://head-server.tf.local/javascript/frontend-log.js?cb=${cb_version}"></script>
+<script src="https://head-server.tf.local/javascript/jquery.js?cb=${cb_version}"></script>
+<script src="https://head-server.tf.local/javascript/bootstrap.js?cb=${cb_version}"></script>
+<script src="https://head-server.tf.local/javascript/select2/select2.js?cb=${cb_version}"></script>
+<script src="https://head-server.tf.local/javascript/spacewalk-essentials.js?cb=${cb_version}"></script>
+<script src="https://head-server.tf.local/javascript/spacewalk-checkall.js?cb=${cb_version}"></script>
+<script src="https://head-server.tf.local/rhn/dwr/engine.js?cb=${cb_version}"></script>
+<script src="https://head-server.tf.local/rhn/dwr/util.js?cb=${cb_version}"></script>
+<script src="https://head-server.tf.local/rhn/dwr/interface/DWRItemSelector.js?cb=${cb_version}"></script>
+<script src="https://head-server.tf.local/javascript/jquery.timepicker.js?cb=${cb_version}"></script>
+<script src="https://head-server.tf.local/javascript/bootstrap-datepicker.js?cb=${cb_version}"></script>
+<script src='https://head-server.tf.local/javascript/momentjs/moment-with-langs.min.js?cb=${cb_version}' type='text/javascript'></script>
```

Comment:
This is really terrible to hardcode a server name... Couldn't we at least use the `https://localhost:8080` that is created by yarn proxy? But then this means that you need to run `yarn proxy --server https://head-server.tf.local` and then `yarn run storybook`.

At least this is working fine for me here.

Reply:
Thanks, It's a good idea! At least we will not need to adapt that file in most of the times. If for some reason we don't want to run the proxy server before we can always adapt that file manually.

This still doesn't solve the problem if we want to deploy the styleguide somewhere, but that will be an effort for another time. To achieve that result we need to either migrate the legacy less->css building of the branding package in OBS to webpack or to somehow inline all the external dependencies inside preview.html. But neither of these 2 solutions will be fast to implement :)